### PR TITLE
Setup Project with Components

### PR DESCRIPTION
**Pull request by V0pe**
### Project Initialization

In the project, I did the following:
- [ ] Add the create app boilerplate
- [ ] Add the route and link for the pages
- [ ] Add the components for the `homepage`

### CODE REVIEW

File: src/components/SingleBook.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function SingleBook() {
+  return (
+    <li>
+      Victor is Victory
+      Victor
+      <button type="button"> Remove </button>
+    </li>
+  );
+}
```
Comment (mmsesay) [inline src/components/SingleBook.js]: - [ ] Let's try to make this component takes in a prop and renders the `title` and `author name`. Doing so will make it dynamic somehow.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @V0pe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

You can also consider:
- [ ] It will be nice to update your README file. Feel free to use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) as an example.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/SingleBook.js (hunk 1 above)
Comment (V0pe) [reply src/components/SingleBook.js]: @mmsesay  Thanks for the insight. I believe you are right. I have done it now

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## To highlights 💪 

- Good job by sending book as props to the SingleBook.js

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure that your README file is descriptive- Feel free to use this [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) as it is the first thing users check.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by V0pe** [comment]
@rindrajosia Thanks for insight. I have adjusted accordingly

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @V0pe  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**